dependencymagnet/doc.go

```diff
@@ -0,0 +1,10 @@
+// +build tools
+
+// dependencymagnet imports code that is not an explicit go dependency, but is used in things
+// like Makefile targets.
+package dependencymagnet
+
+import (
+	_ "github.com/jteeuwen/go-bindata/go-bindata"
+	_ "github.com/openshift/library-go/alpha-build-machinery"
```

Comment:
@gabemontero I think you need another dep magnet for `k8s.io/code-generator`.

See reference in https://github.com/openshift/cluster-samples-operator/blob/master/hack/codegen/update-generated.sh#L7

That said I think we should switch to library-go's alpha-build-machinery in a separate PR.